test/e2eTests/tests/registerPatient_test.js

```diff
@@ -22,28 +39,250 @@ Scenario('Validate tab sequence', (I, RegistrationDashboardPage) => {
+})
+
+Scenario('Register a patient', (I, Data, RegisterPatientPage, RegistrationDashboardPage) => {
+  const patient = Data.patients.patient3;
+
+  I.say(`${LOG_TAG} login`);
+  let dashboardPage = I.login();
+
+  I.say(`${LOG_TAG} Navigate to the registration page`);
+  const registrationPage = dashboardPage.navigateToRegistrationPage();
+
+  I.say(`${LOG_TAG} Click on the New Patient button`);
+  registrationPage.clickNewPatienButton();
+
+  I.say(`${LOG_TAG} Make sure the register patient page is loaded`);
+  RegisterPatientPage.isLoaded();
+
+  validateRequiredFields(I, RegisterPatientPage, 'Identifier', 1);
+  I.say(`${LOG_TAG} Fill in the NID identifier and move to Name tab`);
+  RegisterPatientPage.fillIdentifierForm(patient);
+  I.waitForInvisible('#overlay', 5);
+  RegisterPatientPage.clickNext();
+
+  validateRequiredFields(I, RegisterPatientPage, 'Names', 2);
+  I.say(`${LOG_TAG} Fill in givenName and familyName and move to Gender tab`);
+  RegisterPatientPage.fillNameForm(patient);
+  RegisterPatientPage.clickNext(10);
+
+  validateRequiredFields(I, RegisterPatientPage, 'Gender', 1);
+  I.say(`${LOG_TAG} Select the gender and move to Age tab`);
+  RegisterPatientPage.selectGender(patient);
+  RegisterPatientPage.clickNext();
+
+  validateRequiredFields(I, RegisterPatientPage, 'Birth Date', 1);
+  I.say(`${LOG_TAG} Fill in the birth date and move to Address tab`);
+  RegisterPatientPage.fillBirthDateForm(patient);
+  RegisterPatientPage.clickNext();
+
+  validateRequiredFields(I, RegisterPatientPage, 'Address', 4);
+  I.say(`${LOG_TAG} Fill in the address and move to Contacts tab`);
+  RegisterPatientPage.fillContactForm(patient);
+  RegisterPatientPage.clickNext();
+
+  validateRequiredFields(I, RegisterPatientPage, 'Provenience', 1);
+  I.say(`${LOG_TAG} Select a provenience and move to Testing tab`);
+  RegisterPatientPage.selectProvenience(patient);
+  RegisterPatientPage.clickNext();
+
+  I.say(`${LOG_TAG} Ignore the testing and move to confirmation page`);
+  RegisterPatientPage.clickNext();
+
+  I.say(`${LOG_TAG} Scroll down and confirm patient registration`);
+  I.scrollPageToBottom();
+  I.click(RegisterPatientPage.buttons.confirm);
+  I.waitForInvisible('#overlay', 5);
+
+  I.see(RegisterPatientPage.translate('COMMON_MESSAGE_SUCCESS_ACTION_COMPLETED'));
+  I.wait(1);
+
+  I.say(`${LOG_TAG} Make sure the registration dashboard page loaded`);
+  RegistrationDashboardPage.isLoaded();
+
+  // TODO: This could be a component or a method in the page class
+  // to verify the loaded patient information
+  I.see(patient.identifiers[0].identifier3);
+  I.see(patient.person.names[0].givenName);
+  I.see(patient.person.names[0].familyName);
+})
+
+// Clicks the next button without filling required fields
+const validateRequiredFields = (I, RegisterPatientPage, step, numOfVisibleElements) => {
+  const errorMessageElement = '[ng-message="required"]';
+
+  I.say(`${LOG_TAG} Try to skip the ` + step);
```

Comment:
include step in the comment like this: `... ${step}`

Reply:
Missed that one. Thanks